### DISCUSSION TITLE
Sort files and repeater by drag&drop

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [directus/directus#9971] [Feature Requests]
I think instead of setting a sort field, it would be easier and intuitive enabling by default drag&drop to order files and repeater field types.

**Comment by rijkvanzanten** [comment]
>  it would be easier and intuitive enabling by default drag&drop to order files and repeater field types.

The particular value for the sort needs to be saved somewhere though, and that field key has to be configurable in order to prevent conflicts when you have multiple sort fields 🙂

**Reply by federicocappellotto97** [reply]
Oh yes, it makes sense

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
